foxy/distribution.yaml

```diff
@@ -2003,6 +2003,21 @@ repositories:
       url: https://github.com/swri-robotics/novatel_gps_driver.git
       version: dashing-devel
     status: developed
+  ntpd_driver:
+    doc:
+      type: git
+      url: https://github.com/vooon/ntpd_driver.git
+      version: ros2
+    release:
+      tags:
+        release: release/foxy/{package}/{version}
+      url: https://github.com/vooon/ntpd_driver-release.git
+      version: 2.0.1-1
+    source:
+      type: git
+      url: https://github.com/vooon/ntpd_driver.git
```

Comment:
Please add a top-level LICENSE file to the source repository (per our [review guidelines](https://github.com/ros/rosdistro/blob/master/REVIEW_GUIDELINES.md#types-of-pull-requests)).

Reply:
Done for both branches: ros1 and ros2.

Reply:
Thank you!